EIPS/eip-1.md

```diff
@@ -195,7 +195,7 @@ References to other EIPs should follow the format `EIP-N` where `N` is the EIP n
 
 ## Auxiliary Files
 
-Images, diagrams and auxiliary files should be included in a subdirectory of the `assets` folder for that EIP as follows: `assets/eip-N` (where **N** is to be replaced with the EIP number). When linking to an image in the EIP, use relative links such as `../assets/eip-1/image.png`.
+Images, diagrams and auxiliary files should be included in a subdirectory of the `assets` folder for that EIP as follows: `assets/eip-N` (where **N** is to be replaced with the EIP number). When linking to an image in the EIP, use relative links such as `../assets/eip-1/image.png`. These files must be released into the public domain (or as close as is legally possible) using [CC0](../LICENSE.md). Solidity files must use the SPDX Identifier `CC0-1.0`.
```

Comment:
I like to agree with the requirment for asset file to be CC-0, but only on condition of that the reference implementation don't have to live in `ethereum/EIPs` repository

Reply:
I disagree. We should definitely force authors to use CC0 if they legally can, since it means that users don't have to worry about attribution and all that stuff that just can't apply to blockchains. If they legally can't use CC0, then and **only** then will we allow non-CC0 assets. But we will allow them.

Reply:
I'm afraid that "as close as is legally possible" is opening the door for a fair bit of conflict in the future. Now editors need to verify claims from authors that "this is the best license I can give this!1!!".

Reply:
Well... we don't necessarily need to verify those claims. Depending on how restrictive the license is, we can, well, not approve the PR.